### PR TITLE
Creates the Election documents using PdfMake

### PR DESCRIPTION
**Pull request by tsiegleauq**
This is still WIP.

However, since these are a lot of changes, I probably already missed some features so feel free to test it.

What is currently missing:
Determine how many BallotPapers shall be printed on a page. Everything else should be pretty much done
- print list of candidates (if in voting mode)
- render tables with the Numbers and % (if any published polls)
- different tables according to voting-mode (yn yna voting
- ballot papers for yn yna voting.

While I was on it, I worked on the motion ballot papers a little

### CODE REVIEW

**Comment by tsiegleauq** [comment]
This should be done now.
@emanuelschuetze already tested yesterdays Version a little

I made some examples as usual. Ass you can tell I did not put any effort in picking good names.

[Wahl GroßeTestwahl mit einem langen langen Titel.pdf](https://github.com/OpenSlides/OpenSlides/files/558024/Wahl.GrosseTestwahl.mit.einem.langen.langen.Titel.pdf)

[Wahlgang 2 YN - Wahl über zwei zeilen ... one two one two.pdf](https://github.com/OpenSlides/OpenSlides/files/558025/Wahlgang.2.YN.-.Wahl.uber.zwei.zeilen.one.two.one.two.pdf)

[Wahlgang 4 YN - Wahl über zwei zeilen ... one two one two.pdf](https://github.com/OpenSlides/OpenSlides/files/558026/Wahlgang.4.YN.-.Wahl.uber.zwei.zeilen.one.two.one.two.pdf)

[Wahlgang 5 YN - Wahl über zwei zeilen ... one two one two.pdf](https://github.com/OpenSlides/OpenSlides/files/558027/Wahlgang.5.YN.-.Wahl.uber.zwei.zeilen.one.two.one.two.pdf)

[Wahlgang 6 YN - Wahl über zwei zeilen ... one two one two.pdf](https://github.com/OpenSlides/OpenSlides/files/558028/Wahlgang.6.YN.-.Wahl.uber.zwei.zeilen.one.two.one.two.pdf)

[Wahlgang 1 VOTE Wahl Titel über zwei Zeilen. One Two One.pdf](https://github.com/OpenSlides/OpenSlides/files/558029/Wahlgang.1.VOTE.Wahl.Titel.uber.zwei.Zeilen.One.Two.One.pdf)

[Wahlgang 3 VOTE Wahl Titel über zwei Zeilen. One Two One.pdf](https://github.com/OpenSlides/OpenSlides/files/558030/Wahlgang.3.VOTE.Wahl.Titel.uber.zwei.Zeilen.One.Two.One.pdf)

[Wahlgang 5 VOTE Wahl Titel über zwei Zeilen. One Two One.pdf](https://github.com/OpenSlides/OpenSlides/files/558031/Wahlgang.5.VOTE.Wahl.Titel.uber.zwei.Zeilen.One.Two.One.pdf)

[Wahlgang 7 VOTE Wahl Titel über zwei Zeilen. One Two One.pdf](https://github.com/OpenSlides/OpenSlides/files/558032/Wahlgang.7.VOTE.Wahl.Titel.uber.zwei.Zeilen.One.Two.One.pdf)

@emanuelschuetze 
@normanjaeckel 
This should be ready to review

**Comment by emanuelschuetze** [comment]
I found some minor task which I splitted out in a new issue: #2559

File: openslides/core/static/js/core/pdf.js [hunk 1]
```diff
@@ -15,6 +19,62 @@ angular.module('OpenSlidesApp.core.pdf', [])
...
+                ],
+            };
+        };
+
+        PdfPredefinedFunctions.getBallotLayoutLines = function() {
+            return {
+                hLineWidth: function(i, node) {
+						return (i === 0 || i === node.table.body.length) ? 0 : 0.5;
+				},
+				vLineWidth: function(i, node) {
```
Comment (normanjaeckel) [inline openslides/core/static/js/core/pdf.js]: Indention looks strange.

**Comment by tsiegleauq** [comment]
> Just one stylisic remark
fixed. was 4 whitespaces too much

**Pull request merged**